x/gov/queryable.go

```diff
@@ -8,51 +8,86 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
+// nolint
+const (
+	QueryProposals = "proposals"
+	QueryProposal  = "proposal"
+	QueryDeposits  = "deposits"
+	QueryDeposit   = "deposit"
+	QueryVotes     = "votes"
+	QueryVote      = "vote"
+	QueryTally     = "tally"
+)
+
```

Comment:
All lcd gov changes in `x/gov/queryable.go` and `x/gov/handler.go` etc. must be split into a seperate PR - this PR is already quite full